download_oscar/dod.py

```diff
@@ -13,14 +13,18 @@
 from download_oscar.status import Status
 
 
-def login(user: str, password: str, s: sessions.Session, headers):
+def login(user: str, password: str, s: sessions.Session, headers: Dict[str, str]):
     """Logs in to the base website of the oscar dataset.
 
     Args:
         user (str): User name of the account to login to.
         password (str): Password for the user.
```

Comment:
Down the road we should find a way of not passing the password in plain text here, but I guess this has to be discussed with Huma-Num as well

Reply:
I agree that this looks not very nice since the password is also specified directly when executing the command line tool.
Since the request is made using https the password should be encrypted once it is send over for the actual login at Huma-Num.

Reply:
Oh yes, sorry I agree, this is not a criticism of download_OSCAR, this is more of a note of something we have to discuss and implement with Huma-Num in the future 😅 

If you know of any library or mechanism that we can suggest to them, please do let us know! 😄